pages/committees.tsx

```diff
@@ -11,16 +11,37 @@ const Committees = () => {
     try {
       const response = await fetch("/api/periods/ow-committees");
       const data = await response.json();
-      setCommittees(data);
+
+      const cachedData = JSON.parse(
+        localStorage.getItem("committeesCache") || "[]"
+      );
+
+      if (JSON.stringify(data) !== JSON.stringify(cachedData)) {
+        localStorage.setItem("committeesCache", JSON.stringify(data));
+        setCommittees(data);
+      } else {
+        setCommittees(cachedData);
+      }
       console.log(data);
     } catch (error) {
       console.error("Failed to fetch committees:", error);
+      const cachedData = JSON.parse(
+        localStorage.getItem("committeesCache") || "[]"
+      );
+      setCommittees(cachedData);
     } finally {
       setIsLoading(false);
     }
   };
 
   useEffect(() => {
+    const cachedData = JSON.parse(
+      localStorage.getItem("committeesCache") || "[]"
+    );
+    if (cachedData.length > 0) {
+      setCommittees(cachedData);
+      setIsLoading(false);
+    }
```

Comment:
cachen må utdateres etterhvert, hvis ikke kan ikke komiteene oppdatere dataen sin.

Reply:
Den oppdateres for hver gang i bakgrunnen, så dersom det skjer noen endringer. Vil det endre seg etter den har hentet komiteer

Reply:
![image](https://github.com/user-attachments/assets/276780a7-a886-48ff-8c3e-4d39005f4ee1)
![image](https://github.com/user-attachments/assets/e7c2b662-a467-41e7-90d6-59a5bc745b7d)
![image](https://github.com/user-attachments/assets/ce2f3b07-4b33-434b-bd58-3d40ea5a000e)

Reply:
Testet med å gjøre noe endringen for appkom på OW, og endringen tok typ 5 sekunder

Reply:
Så cachingen skal være good🚀